src/main/src/main/java/org/geoserver/catalog/impl/StoreInfoImpl.java

```diff
@@ -203,4 +205,14 @@ public boolean equals(Object obj) {
         } else if (!workspace.equals(other.getWorkspace())) return false;
         return true;
     }
+
+    @Override
+    public Date getDateModified() {
+        return getMetadata().get(PublishedInfo.TIME_MODIFIED, Date.class);
```

Comment:
This is repeat everywhere and then, don't this class have a common interface? e.g ``CatalogInfo``?

Reply:
agreed will move it to CatalogInfo